junit-jupiter-api/src/main/java/org/junit/jupiter/api/IndicativeSentencesSeparator.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2015-2019 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * https://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junit.jupiter.api;
+
+import static org.apiguardian.api.API.Status.EXPERIMENTAL;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.apiguardian.api.API;
+
+/**
+ * {@code @IndicativeSentencesSeparator} is used to declare a custom separator
+ * in the IndicativeSentencesGenerator, if this notation was not declared
+ * IndicativeSentencesGenerator will use ", " as separator by default.
+ *
+ * @since 5.6
+ * @see DisplayNameGenerator
+ */
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+@Inherited
+@API(status = EXPERIMENTAL, since = "5.6")
+public @interface IndicativeSentencesSeparator {
+
+	/**
+	 * Custom separator for indicative sentences generator.
+	 *
+	 * @return custom separator for indicative sentences
+	 */
+	String value();
```

Comment:
Let's set `""` as the dummy default value so users can use the annotation without any parameters and we can detect whether they set the separator explicitly.

Reply:
Okay, I hadn't thought about the case of the annotation without the parameter.